src/cpu/x64/brgemm/capi/brgemm_api.cpp

```diff
@@ -24,6 +24,7 @@
 
 #include "cpu/x64/brgemm/brgemm.hpp"
 
+#include <memory>
```

Comment:
This is already included since `utils` `namespace` is available.

Reply:
thanks for the feedback, will update